### ISSUE TITLE
Implement "e-ink mode" for paginated swiping

### ISSUE THREAD

**Issue by chocolatkey** [readium/ts-toolkit#25]
In this mode, when you swipe, instead of the page offset following your finger movements, the reader pops to the next/previous column

**Comment by mickael-menu** [comment]
This setting could also be useful beyond e-ink, for example accessibility (see `prefers-reduced-motion` CSS media feature). So we could use a more generic name, like `transition=false`.

**Comment by chocolatkey** [comment]
Yes, I have the same thing in my manga reader, an animations toggle that disables all page turn/move animations (in addition to ui animations). Even people without accessibility impairments like having this for e.g. lag purposes

**Comment by JayPanoz** [comment]
This popped up again this morning while discussing media query `prefersReducedMotion` – that is already supported and used in Readium Playground (for bottom sheet transitions on snap points) – as an aside of another detail of the Preferences API migration. 

Given the use case of having a user-setting for it has already been mentioned and is a fairly common setting, I guess this is bound to be added to the Preferences and Settings (Preferences API) at some point.

In any case the scope would be larger than swiping, and applies to all means of navigation (we have an `animated` argument  for methods but then we need to define how conflicts should be resolved, etc.).
